### PR TITLE
Add reference to Ogham Spring Boot starters

### PR DESCRIPTION
**Pull request by aurelien-baudet**
I would like to include Ogham in Spring Boot official starters.

The aim of Ogham is to simplify the code that sends email and SMS (more is coming like RCS).
You write the same code but still target different protocol/services (for example SMTP in dev, SendGrid in production).
You automatically use the template engine that is available in the classpath to design the content of your email or your SMS.
You also get rid of technical concerns (for example, images and styles are automatically inlined in the HTML of the email, you don't need to do it).
You also increase productivity while testing using testing tools.

There are more features, see [user manual](https://groupe-sii.github.io/ogham/v3.0.0/user-manual.html).

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@aurelien-baudet Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-boot&pullRequestId=24964)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-boot&pullRequestId=24964) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

File: spring-boot-project/spring-boot-starters/README.adoc [hunk 1]
```diff
@@ -241,4 +241,9 @@ do as they were designed before this was clarified.
 | https://github.com/knowm/XChange[XChange]
 | https://github.com/cassandre-tech/cassandre-trading-bot
 
+| https://groupe-sii.github.io/ogham/[Ogham]
```
Comment (wilkinsona) [inline spring-boot-project/spring-boot-starters/README.adoc:244]: The list of starters is ordered alphabetically. Could you please move this entry further up the page so it comes immediately before the entry fo OkHTTP.

File: spring-boot-project/spring-boot-starters/README.adoc [hunk 2]
```diff
@@ -241,4 +241,9 @@ do as they were designed before this was clarified.
...
+a| * https://github.com/groupe-sii/ogham/tree/master/ogham-spring-boot-starter-all
```
Comment (wilkinsona) [inline spring-boot-project/spring-boot-starters/README.adoc:245]: For consistency with the other entries, could you please remove the bullet points?

**Review by wilkinsona — CHANGES_REQUESTED** [review]
Thanks very much for the PR, @aurelien-baudet. I've made a couple of suggestions. Could you please take a look at them when you have a moment?

**Comment by wilkinsona** [comment]
@aurelien-baudet Thanks very much for making your first contribution to Spring Boot.

**Pull request closed without merging**